library/src/main/java/me/toptas/fancyshowcase/FancyShowCaseView.java

```diff
@@ -411,6 +464,35 @@ public Builder titleStyle(@StyleRes int style, int titleGravity) {
         }
 
         /**
+         * @param titleGravity title gravity
+         * @return Builder
+         */
+        public Builder titleGravity(int titleGravity) {
+            mTitleGravity = titleGravity;
+            return this;
+        }
+
+        /**
+         * @param titleSize title size, overrides any defined size in the default or provided style
+         * @return Builder
+         */
+        public Builder titleSizePx(int titleSize) {
```

Comment:
I've seen this pattern quite often already, I personally prefer this. I can change this though if you really want to.